### DISCUSSION TITLE
tlsSidecarEntityOperator trying to start with "kafka-3.4.0"

### DISCUSSION THREAD

**Discussion by nazar200** [strimzi/strimzi-kafka-operator#8126] [Q&A] [answered]
In my stack, I am using kafka 3.2.0 and we are using helm to deploy it.
I have pushed the kafka image 3.2.0 in my private repo. 
But when I try to install operator 33.1 , tlsSidecarEntityOperator pod not coming up saying image not found.
and found its trying to download Kafka image "kafka-3.4.0, So want to know why its always trying to use kafka.3.4.0

**Comment by scholzj** [comment]
You are most likely not using 0.33.1 if it uses 3.4.0 images. But in
general, the TLS sidecar is bot Kafka based. So it does not allign the
Kafka version you use. It will be always the qltest Kafka version supported.

**Reply by nazar200** [reply]
Okay.. thanks @scholzj ,I have tested both 33.1 and 33.2. 
Exporter also uses the different kafka image,  hope this won't impact the metrics for the kafka_version we use (3.2.0)

**Comment by scholzj** [comment, marked as answer]
As I said, that is by design. They do not use anything from the Kafka
binaries.
